### DISCUSSION TITLE
Export models to Tensorflow or ONNX

### DISCUSSION THREAD

**Discussion by neelakanth** [flashlight/flashlight#712] [Q&A]
Hi,
I am looking for examples on how to export models trained with flashlight to tensorflow or ONNX for inference.

Regards,
Neelakanth

**Comment by tlikhomanenko** [comment]
Hey, we didn't do specific scripts. In our practice we did for each particular model its own conversion. At the end model is just weights, so you dump weights into, say, txt file from flashlight and then load in a proper manner in tensorflow with proper model definition.
